packages/cli/src/lib/core/prompt/confirm-to-process.unit.test.ts

```diff
@@ -0,0 +1,71 @@
+import { confirmToProcess } from './confirm-to-process';
+import { RcJson } from '../../types';
+import prompt = require('./prompt');
+
+jest.mock('./prompt');
+
+describe('confirmToProcess', () => {
+  const CONTEXT = {} as RcJson;
+  const mockPrompt = 'Confirm should process?';
+  const promptParamSpy = jest.spyOn(prompt, 'promptParam');
+  const mockProcess = jest.fn();
```

Comment:
Personally, I would prefer having the `jest.fn()` directly in the test unless explicitly checking the process call (last two tests) in order to have all the context inside each test clearly stated.

Reply:
Great agreed